packages/server/lib/plugins/child/ts_node.js

```diff
@@ -46,6 +46,18 @@ const getTsNodeOptions = (tsPath, registeredFile) => {
   const opts = {
     compiler, // use the user's installed typescript
     compilerOptions,
+    ignore: [
+      // default ignore
+      '(?:^|/)node_modules/',
+      // do not transpile cypress resources
+      // getIgnoreRegex({ configDir: dir, currentFileDir: __dirname, sep: path.sep }),
+      // This is not ideal, We are transpiling any pre-built cypress code along with the config file
+      // Ideally we'd only transpile the config file but deriving the correct has proven to be tricky
+      // due to differences between dev and prod, and quirks of ts-node's path handling
+      // We do not want to ignore too much or to little
+      // So for now I am only ignoring the explicit file that has issues
```

Comment:
to keep the language consistent with above
```suggestion
      // So for now we are only ignoring the explicit file that has issues
```